kea-vscode/src/views/common/file-tree-node.ts

```diff
@@ -30,16 +32,33 @@ export class FileTreeNode implements IParentTreeNode<ReviewCommentTreeNode> {
       fileStatus: file.status,
     });
 
+    // Create a file URI for icon purposes.
+    this.#fileUri = vscode.Uri.file(file.filename);
+
     this.#comments = comments;
```

Comment:
Comments should be common to both the `LocalFileTreeNode` and this. Make sure that `LocalFileTreeNode` can get comments for its files.